### PR TITLE
Add SciPy 2022 talk to presentations

### PR DESCRIPTION
**Pull request by maxrjones**
**Description of proposed changes**

Adds a link to the SciPy talk in the overview page.

**Preview** at https://pygmt-dev--2053.org.readthedocs.build/en/2053/overview.html#presentations

Fixes #

**Reminders**

- [ ] Run `make format` and `make check` to make sure the code follows the style guide.
- [ ] Add tests for new features or tests that would have caught the bug that you're fixing.
- [ ] Add new public functions/methods/classes to `doc/api/index.rst`.
- [ ] Write detailed docstrings for all functions/methods.
- [ ] If wrapping a new module, open a 'Wrap new GMT module' issue and submit reasonably-sized PRs.
- [ ] If adding new functionality, add an example to docstrings or tutorials.

**Slash Commands**

You can write slash commands (`/command`) in the first line of a comment to perform
specific operations. Supported slash commands are:

- `/format`: automatically format and lint the code
- `/test-gmt-dev`: run full tests on the latest GMT development version

### CODE REVIEW

**Review by weiji14 — APPROVED** [review]
Nicely done, thanks again @maxrjones for giving the presentation! It's just outstanding to see how far PyGMT has come along since 2017 :smile:

**Comment by seisman** [comment]
Nice presentation! 

For this PR, I feel the thumbnail image `doc/_static/scipy2022-youtube-thumbnail.png` is too large (834 KB) and should not be included in the repo.

**Review by michaelgrund — APPROVED** [review]

**Comment by seisman** [comment]
> For this PR, I feel the thumbnail image `doc/_static/scipy2022-youtube-thumbnail.png` is too large (834 KB) and should not be included in the repo.

The thumbnail image can be compressed to less than 100 KB using https://squoosh.app/.

File: doc/overview.rst [hunk 1]
```diff
@@ -41,6 +41,18 @@ Presentations
 These are conference presentations about the development of PyGMT (previously
 "GMT/Python"):
 
+* "Geospatial Analysis & Visualization with PyGMT".
+  2022.
+  Max Jones, Wei Ji Leong, and Leonardo Uieda.
+  Presented at *SciPy 2022*.
+  doi:`10.6084/m9.figshare.20483793 <https://doi.org/10.6084/m9.figshare.20483793>`__
+
+.. figure:: _static/scipy2022-youtube-thumbnail.png
```
Comment (weiji14) [inline doc/overview.rst:50]: > Nice presentation!
> 
> For this PR, I feel the thumbnail image `doc/_static/scipy2022-youtube-thumbnail.png` is too large (834 KB) and should not be included in the repo.

Could we source the thumbnail from Youtube directly (i.e. use a relative link)? E.g. following https://stackoverflow.com/questions/14192709/is-it-possible-to-embed-youtube-vimeo-videos-in-markdown-using-a-c-sharp-markdow. Downside is that places blocking Youtube might not be able to see the thumbnail :slightly_smiling_face:

Comment (seisman) [reply doc/overview.rst:50]: The thumbnail image (https://img.youtube.com/vi/nCktihu9bWg/0.jpg) is not as good as the screenshot.

Comment (maxrjones) [reply doc/overview.rst]: I followed the recommendation to compress using https://squoosh.app/. Thanks for the suggestion.

**Review by seisman — APPROVED** [review]

**Pull request merged**